code/Controllers/ContentController.php

```diff
@@ -139,6 +140,14 @@ protected function init()
     {
         parent::init();
 
+        // In the CMS Preview or draft contexts, we never want to cache page output.
+        if (
+            $this->getRequest()->getVar('CMSPreview') === "1"
+            || $this->getRequest()->getVar("stage") === "Stage"
```

Comment:
Any reason for the mixture of quote types? We usually stick to single unless we need double.

You can also use `Versioned::DRAFT` instead of `"Stage"` if you'd like to use a constant.

Reply:
Tidied, sorry! Hopefully nabbed the PSR-2 failure too (FWIW I was following PSR-12, didn't expect it to be different).